libs/state-machine/src/lib/getStepList.ts

```diff
@@ -26,16 +29,25 @@ export const getStepperConfig = (wizard: WizardLayout): StepperConfig => {
 					parentStepId: step.parentStepId,
 					canSkipTo: step.canSkipTo,
 					canSkipFrom: !!step.executeSkip,
+					schema: step.schema,
 				},
 			];
 		},
 		[],
 	);
 
+	// TO DO - these should really be properties of the WizardLayout, rather than
```

Comment:
If you rename TO DO as TODO I think vscode picks it up as a work list with an extension

![image](https://user-images.githubusercontent.com/4472313/236494607-a12015f2-f9dc-4592-9b85-a8ec065d388a.png)

Reply:
Thanks!  we have a quite a lot of "TO DO"'s in the repo - I'll add a heath ticket to make them all "TODO". 

```
// TODO make this TO DO a TODO
// TO DO - fix this
```